models/product/src/types.ts

```diff
@@ -0,0 +1,29 @@
+import { TReferenceGraphql } from '@commercetools-test-data/commons';
+import type { TBuilder } from '@commercetools-test-data/core';
+import { Product, ProductDraft } from '@commercetools/platform-sdk';
+
+export type TProduct = Product & {
```

Comment:
Do we need a REST type that doesn't have `skus`?

Reply:
YES! Should've been in the previous commit. Fixed with [7320171](https://github.com/commercetools/test-data/pull/219/commits/7320171b756be7ae071b24746474542739934d28)